api/types/app_test.go

```diff
@@ -164,3 +164,53 @@ func TestAppServerSorter(t *testing.T) {
 	servers := makeServers(testValsUnordered, "does-not-matter")
 	require.True(t, trace.IsNotImplemented(AppServers(servers).SortByCustom(sortBy)))
 }
+
+func TestAppIsAWSConsole(t *testing.T) {
+	tests := []struct {
+		name         string
+		uri          string
+		isAWSConsole bool
+	}{
+		{
+			name:         "AWS Standard",
+			uri:          "https://console.aws.amazon.com/ec2/v2/home",
+			isAWSConsole: true,
+		},
+		{
+			name:         "AWS China",
+			uri:          "https://console.amazonaws.cn/console/home",
+			isAWSConsole: true,
+		},
+		{
+			name:         "AWS GovCloud (US)",
+			uri:          "https://console.amazonaws-us-gov.com/console/home",
+			isAWSConsole: true,
+		},
+		{
+			name:         "Region based not supported yet",
+			uri:          "https://us-west-1.console.aws.amazon.com",
+			isAWSConsole: false,
+		},
+		{
+			name:         "Not an AWS Console URL",
+			uri:          "https://hello.world",
+			isAWSConsole: false,
```

Comment:
nit: If you change the type to `require.BoolAssertionFunc` and assign na assert function instead of `bool`, then you can remove the `if` below in the test.